### PR TITLE
Use `pkg:lints` and `pkg:flutter_lints` re-build

### PR DESCRIPTION
**Pull request by domesticmouse**
Re-manufacturing #706, because reasons.

fixes: https://github.com/dart-lang/dart-pad/issues/1881

### CODE REVIEW

**Review by srawlins — APPROVED** [review]
Love it

File: lib/src/common.dart [hunk 1]
```diff
@@ -189,11 +187,11 @@ class _MyHomePageState extends State<MyHomePage> {
         child: Column(
           mainAxisAlignment: MainAxisAlignment.center,
           children: <Widget>[
-            Text(
+            const Text(
               'You have pushed the button this many times:',
             ),
             Text(
-              '$_counter',
+              _counter.toString(),
```
Comment (mit-mit) [inline lib/src/common.dart:194]: I don't think the lints require this change. Isn't it better to keep this aligned with the `flutter create` template?

Comment (domesticmouse) [reply lib/src/common.dart:194]: Without that change the variable is marked as unused. I only made the changes required to get the tests to pass.

Comment (mit-mit) [reply lib/src/common.dart:194]: That sounds unexpected. Got a log or some repro steps for that issue?

**Comment by mit-mit** [comment]
I landed my PR, so the diff here is now much smaller.

File: lib/src/common.dart (hunk 1 above)
Comment (domesticmouse) [reply lib/src/common.dart]: Turns out I was holding it wrong, raw strings to the rescue.

**Review by mit-mit — APPROVED** [review]
Thanks Brett!

**Comment by domesticmouse** [comment]
Blocked on https://github.com/flutter/codelabs/pull/134 and https://github.com/flutter/samples/pull/804

**Comment by domesticmouse** [comment]
@RedBrogdon What else is required for this PR to land?

**Comment by mit-mit** [comment]
@RedBrogdon @domesticmouse can we merge this now?

**Comment by domesticmouse** [comment]
> @RedBrogdon @domesticmouse can we merge this now?

Hey @mit-mit, we are going to land this early next week. There will be some samples in various places that will have analysis warnings we this lands, and we want @RedBrogdon at the ready to push updates to the docs as required.

**Comment by domesticmouse** [comment]
Landing this on master now. I'll switch over the servers tomorrow.

**Pull request merged**